docs/2_3_common_states.adoc

```diff
@@ -1,63 +1,68 @@
 === State Machine and Semantics [[common-state-machine]]
 
+To define allowed calling sequences of FMI functions, state machines are used.
 All FMI interface types share a number of states in their respective state machines.
-This chapter describes these common modes.
-FMI specific state-machine modes will be described in their respective chapters.
+This chapter describes these common states used in at least two of the interface types.
+FMI specific state-machine states will be described in their respective chapters.
+Each state of the state machine corresponds to a certain phase of a simulation.
 
 .Common calling sequence for C functions of common states for all three FMI types.
 [#figure-common-state-machine]
 image::images/state-machines-common-states.svg[width=80%, align="center"]
 
-==== Super State: FMU State Setable [[FMUStateSetable]]
+The state machine is given here as UML 2.0 state machine.
+If a transition is labeled with one or more function names (for example, <<fmi3EnterInitializationMode>>, <<fmi3EnterEventMode>>), the transition is taken if the function call returns successfully (not `NULL` for <<fmi3Instantiate>> or <<fmi3OK>> and <<fmi3Warning>> for all other functions).
+This way, importer and FMU can determine in which state the FMU is.
 
-The description of the super state *FMU State Setable* generally describes functions that deal with instantiation, destruction and logging of FMUs.
+==== Super State: FMU State Setable [[FMUStateSetable,*FMU State Setable*]]
 
-This state is entered when any of the following functions is called: <<fmi3InstantiateModelExchange>>, <<fmi3InstantiateCoSimulation>> and <<fmi3InstantiateScheduledExecution>>.
-The state is left by either calling <<fmi3FreeInstance>> or when any of the functions called during *FMU State Setable* returns <<fmi3Fatal>>.
-If any function called in super state *FMU State Setable* returns <<fmi3Error>>, the FMU enters state *Terminated*.
+The state <<FMUStateSetable>> is entered when any of the following functions is called: <<fmi3InstantiateModelExchange>>, <<fmi3InstantiateCoSimulation>> and <<fmi3InstantiateScheduledExecution>>.
+The state is left by either calling <<fmi3FreeInstance>> or when any of the functions called during <<FMUStateSetable>> returns <<fmi3Fatal>>.
+If any function called in super state <<FMUStateSetable>> returns <<fmi3Error>>, the FMU enters state <<Terminated>>.
 
 Allowed Function Calls::
 
 [[fmi3InstantiateModelExchange,`fmi3InstantiateModelExchange`]]
```

Comment:
We debated this back and forth - no matter how you do it, it breaks the pattern.
We decided that have it this way so it could be found more easily.
The anonymous start state cannot really be listed: it has no name and if you read the state chart, you can hardly see it.
Unless you are a comp-sci professional, start state is an unknown concept.

Simply moving it before the state "FMU state setable" hides it when jumping to "FMU state setable".
To list it outside the "Allowed Function Calls" might also be confusing when we convert the Function XXX:: to headings (still discussed now and then).

Conclusion: this is not an oversight, this is a compromise and we have not been able to find a better solution.

Reply:
I suggest  to create here a special section:
Allowed Function Calls to enter this Super State.

Reply:
Fixed as proposed.